struts2-jquery-plugin/.gitignore

```diff
@@ -56,3 +56,11 @@ src/main/resources/template/js/base/widget.min.js
 /src/main/resources/template/js/base/checkboxradio.min.js
 /src/main/resources/template/js/base/form-reset-mixin.min.js
 /src/main/resources/template/i18n/jquery-ui-timepicker-*.min.js
+
```

Comment:
@jogep 
My mistake. i must have accidently ignored the full files instead of the minified versions.

I'll fix it ASAP